README.md

```diff
@@ -14,6 +14,34 @@ The boilerplate contains the following features:
 
 Currently the Ragnarok boilerplate is synchronized with the Laravel 8.0.3 starter template. 
 
+## How To Use
+
+This project is using [Laravel 8.*](https://laravel.com/docs/8.x) and Database MySQL.
+Several things need to setup like creating the key, setup the environment database login, etc.
+
+Here is how to use this project step by step:
+- After you clone this repo, don't forget to do (Make sure you install composer first, refer [here](https://getcomposer.org)):
+```composer install``
+- Setup your environment, copy the .env.example to .env, and filling those credential database.
+- After you do the install composer, do the following ```php artisan key:generate```
+- Then do migrate and seeding the database with ```php artisan migrate:refresh --seed```
+- After you done those steps, time to serving the laravel by using ```php artisan serve```
+- Then go to your browser (by default it would using port 8000), so you can access with ```localhost:8000/home```
+
+## Login Credential
```

Comment:
Maybe it is a better wording choice if we replace `Login Credential` to `Default login credentials`

Reply:
And also u forked from the main branch. Which didn't include the logo that is added in the develop branch

Reply:
Okay, I would fix it. And I just saw there are 2 branch, main and develop. 
Thank you for raising those issue.